docs/extra/PubisherClientSnippets.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Will leave for now - this file will be deleted in the next PR anyway, when a single file (snippets.md) will be generated for everything.